### PR TITLE
DOC Update README.md for CMS 5

### PR DESCRIPTION
**Pull request by GuySartorelli**
## Issue

- https://github.com/silverstripe/.github/issues/20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,10 @@
...
 ## Installation
-```
-$ composer require --dev silverstripe/graphql-devtools
-```
-
-## Requirements
 
-* [silverstripe/graphql](https://github.com/silverstripe/silverstripe-graphql)
+```sh
+composer require silverstripe/graphql-devtools
```
Comment (sabina-talipova) [inline README.md]:
```suggestion
composer require --dev silverstripe/graphql-devtools
```

File: README.md
Comment (sabina-talipova) [inline README.md:1]: https://github.com/silverstripe/silverstripe-graphql-devtools/blob/b8decf3b574434d68f75cc80bf443f30cb6b59aa/README.md?plain=1#L41

Could you, please, replace double dashes with one dash.

**Review by sabina-talipova — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ChloeSartorelli) [reply README.md]: Done

File: README.md
Comment (ChloeSartorelli) [reply README.md:1]: Done

**Review by sabina-talipova — APPROVED** [review]
LGTM

**Pull request merged**